lib/src/server_messages.dart

```diff
@@ -158,16 +158,33 @@ class NotificationResponseMessage extends ServerMessage {
 class CommandCompleteMessage extends ServerMessage {
   final int rowsAffected;
 
-  static RegExp identifierExpression = RegExp(r'[A-Z ]*');
+  /// Match the digits at the end of the string.
+  /// Possible values are:
+  ///  ```
+  ///  command-tag | #rows
+  ///  SELECT 1
+  ///  UPDATE 1234
+  ///  DELETE 568
+  ///  MOVE 42
+  ///  FETCH 60
+  ///  COPY 314
+  ///  ```
+  ///  For INSERT, there are three columns:
+  ///  ```
+  ///  | command tag | oid* | #rows |
+  ///  INSERT 0 42
+  ///  ```
+  ///  *oid is only used with `INSERT` and it's always 0.
+  static RegExp identifierExpression = RegExp(r'\d+$');
```

Comment:
This is no longer `identifierExpression`, rather `affectedRowsExpression`. Maybe call it `_affectedRowsExp` to be short and also private?

Reply:
Done